test/Microsoft.AspNet.Razor.Runtime.Test/TagHelpers/TagHelperDescriptorResolverTest.cs

```diff
@@ -149,7 +149,7 @@ public static TheoryData ResolveDirectiveDescriptorsInvalidTagHelperPrefixData
                             },
                             new TagHelperDirectiveDescriptor
                             {
-                                DirectiveText = "*Plain*, " + assemblyA,
+                                DirectiveText = "Microsoft.AspNet.Razor.Runtime.TagHelpers.Valid_P*, " + assemblyA,
```

Comment:
Of course, I'm more or less just asking if we should delete these cases entirely instead of repurposing to further test `___*`.